README.md

```diff
@@ -56,6 +56,23 @@ require_once dirname(__FILE__).'/omise-php/lib/Omise.php';
 
 Please see usage section below for usage examples.
 
+### API version
+
```

Comment:
To choose an specific API version to use, you can specify it by setting the api_version. 
The version configured here will override the version set in your account.
This is useful if you have multiple environments with different API versions for testing.
(e.g. development on the latest but production on the older version).